docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -49,11 +51,24 @@ \section{Team Member Roles}
 
 \section{Workflow Plan}
 
-\begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classificaiton, etc.?
-\end{itemize}
+We will be using git as a version control software with GitHub as our remote repository. The root branch will be \code{master} and development branch mased on \code{master} will be \code{dev}. The developer will branch off from \code{dev} to work on issues. There will be a new branch for each issue and will be named comtaining the issue number and name detailed below. Continuous Inntegration is set up between all branches \code{dev}, \code{master}, and Issues.
```

Comment:
- We will be using git as a version control software with GitHub as our remote repository. The root branch will be \code{master} and development branch _based_ on \code{master} will be \code{dev}.
- Continuous _Integration_ is set up between all branches \code{dev}, \code{master}, and Issues.